src/components/ShowMore/ShowMore.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { SyntheticEvent } from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+
+type Props = {
+  className?: string;
+  label: string;
+  onClick: (event: SyntheticEvent) => void;
```

Comment:
Initially as "ShowMore" component the label and click were required (as they were minimal things needed to render), but if we treat it more as "SummaryButton" (summary with an action), this could be made optional.